compositor_render/src/transformations/web_renderer/mod.rs

```diff
@@ -0,0 +1,8 @@
+mod electron_api;
+pub mod transformation;
+
+pub use transformation::*;
+
+type Url = String;
+
+type SessionId = String;
```

Comment:
check https://github.com/membraneframework/video_compositor/pull/22#discussion_r1271925086 - similiar reasoning applies in this case.

I would suggest:
- SessionId either keep as just a String or create new type if you want to have a better type safety
- Url either keep String or maybe check for some already existing Url representations in rust.